infrastructure/main.tf

```diff
@@ -9,11 +9,14 @@ locals {
 }
 
 module "db" {
-  source              = "git@github.com:hmcts/moj-module-postgres"
-  product             = "${var.product}-db"
+  source              = "git@github.com:hmcts/moj-module-postgres?ref=cnp-449-tactical"
+  product             = "${var.product}-postgres-db"
```

Comment:
I had to change the name - otherwise the deployment would complain about the server already existing.